src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -383,11 +383,15 @@ add_subset_sensitivity(TargetT& sensitivity, const int subset_num) const
           view <= proj_data_info_sptr->get_max_view_num();
           view += this->num_subsets)
       {
-        const ViewSegmentNumbers view_segment_num(view, segment_num);
+        for (int timing_pos_num = proj_data_info_sptr->get_min_tof_pos_num();
+            proj_data_info_sptr->get_max_tof_pos_num(), timing_pos_num++;)
+        {
+          const ViewSegmentTOFNumbers view_segment_num(view, segment_num, timing_pos_num);
```

Comment:
this would need more work. In the original code, there is some logic to avoid doing TOF backprojs in the sensitivity calculation (unless forced to).

Reply:
probably need to check if this stuff is still used though (we have `distributable_sensitivity`)

Reply:
Related to https://github.com/UCL/STIR/issues/997 ?

Reply:
yes